posthog/api/plugin.py

```diff
@@ -285,7 +286,7 @@ def filter_queryset_by_parents_lookups(self, queryset):
 
     @action(methods=["GET"], detail=False)
     def repository(self, request: request.Request, **kwargs):
-        url = "https://raw.githubusercontent.com/PostHog/integrations-repository/main/plugins.json"
+        url = "https://raw.githubusercontent.com/PostHog/apps/main/manifest.json"
```

Comment:
I think this is the correct location, though `manifest.json` sounds misleading. I'm not a linguist, but from my common understanding, an `app manifest` is something that's connected to a single app, not a list of apps. Thus I'd propose calling this `repository.json`.

Reply:
Agreed, especially since we call it repository within the app :+1:

Reply:
Updated :heavy_check_mark: